design/video-infrastructure.md

```diff
@@ -2,49 +2,49 @@
 
 ![](./assets/video-infrastructure.webp)
 
-When a streamer goes live they hit the first microservice called ingest.
+When a streamer goes live, they hit the first microservice called ingest.
 
 ## Ingest
 
-Ingest is responsible for handling the incoming go live requests. 
+Ingest is responsible for handling the incoming go-live requests. 
 
-It is agnostic to the protocol used to go live, for example we can support:
+It is agnostic to the protocol used to go live; for example, we can support the following:
```

Comment:
"It is agnostic to the protocol used for the video stream;" maybe.

Reply:
`It is agnostic to the protocol used by the streamer to go live`